lib/CSkillHandler.cpp

```diff
@@ -182,10 +168,15 @@ CSkill * CSkillHandler::loadFromJson(const JsonNode & json, const std::string &
 		for(auto b : levelNode["effects"].Struct())
 		{
 			auto bonus = JsonUtils::parseBonus(b.second);
-			bonus->sid = skill->id;
 			skill->addNewBonus(bonus, level);
 		}
-		skill->setDescription(levelNode["description"].String(), level);
+		skill->at(level).description = levelNode["description"].String();
+		if(!levelNode["images"].isNull())
```

Comment:
This check not needed, you can call `levelNode["images"]["small"].String()` on Null node. But you could change it to check for `JsonType::DATA_STRUCT`